src/Bindings/ManualBindings.cpp

```diff
@@ -2823,64 +2823,56 @@ class cLuaBlockTracerCallbacks :
 	{
 	}
 
-	virtual bool OnNextBlock(int a_BlockX, int a_BlockY, int a_BlockZ, BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, eBlockFace a_EntryFace) override
+	virtual bool OnNextBlock(Vector3i a_Block, BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, eBlockFace a_EntryFace) override
 	{
 		bool res = false;
-		if (!m_Callbacks->CallTableFn(
-			"OnNextBlock",
-			a_BlockX, a_BlockY, a_BlockZ, a_BlockType, a_BlockMeta, a_EntryFace,
-			cLuaState::Return, res
-		))
+		if (
+			m_Callbacks->CallTableFn("OnNextBlock", a_Block, a_BlockType, a_BlockMeta, a_EntryFace, cLuaState::Return, res)
+		)
```

Comment:
The linebreaks now make little sense, either make this all a single line, or break the args apart.

Reply:
It is now break apart.